### PR TITLE
Refresh Icon For FilterList

### PR DESCRIPTION
**Pull request by dshishkov**
reference https://github.com/smartprocure/bid-search/issues/857
reference https://github.com/smartprocure/bid-search/issues/909

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 642e48a896ec8155651ca3b69ae46cc3a8a3ecff
</p>

File: src/DefaultIcon.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ let iconMap = {
   PreviousPage: () => <span>←</span>,
   Previous5Pages: () => <span>⇜</span>,
   Next5Pages: () => <span>⇝</span>,
+  Refresh: ({ onClick }) => <span onClick={onClick}>⟳</span>,
```
Comment (daedalus28) [inline src/DefaultIcon.js]: In other places, we’ve just wrapped the icon with whatever which handles the onClick (e.g. using IconButton in grey vest or even just a div). We should probably stay consistent with that pattern

File: src/FilterList.js [hunk 2]
```diff
@@ -23,6 +26,23 @@ export let Label = inject(_.pick('tree'))(
       {tree &&
         node && (
           <span className="filter-field-label-icon">
+            {!node.updating &&
+              !!node.hasValue &&
```
Comment (daedalus28) [inline src/FilterList.js]: It shouldn’t matter if the current node has a value

File: src/FilterList.js [hunk 3]
```diff
@@ -23,6 +26,23 @@ export let Label = inject(_.pick('tree'))(
...
+              tree.disableAutoUpdate &&
+              // find if any nodes in the tree are marked for update (i.e. usually nodes are marked for update because they react to "others" reactor)
+              _.some(
```
Comment (daedalus28) [inline src/FilterList.js]: Did you try just checking tree.markedForUpdate? That should probably do it I think

Comment (dshishkov) [reply src/FilterList.js]: @daedalus28 I thought so too but tree.markedForUpdate remains false no matter how many times I update other filter nodes. Not sure if this is a contexture-client bug or something else?

Comment (daedalus28) [reply src/FilterList.js]: Actually, that makes sense, this part is probably fine. We should consider adding something for this in contexture-client but this is ok for now

File: src/DefaultIcon.js (hunk 1 above)
Comment (dshishkov) [reply src/DefaultIcon.js]: done

File: src/FilterList.js (hunk 2 above)
Comment (dshishkov) [reply src/FilterList.js]: removed the hasValue check

**Pull request merged**